lib/github/graphql.js

```diff
@@ -0,0 +1,37 @@
+module.exports = addGraphQL
+
+function addGraphQL (octokit) {
+  octokit.query = query.bind(null, octokit)
+}
+
+async function query (octokit, query, variables = undefined) {
```

Comment:
Could we add a `headers = {}` here so I can pass some special github staff headers here? It will make testing hubot against feature flagged items way easier.